system/LINK/etc/default/evm

```diff
@@ -22,7 +28,17 @@ export QPID_LOAD_MODULE=/usr/lib64/qpid/client/sslconnector.so
 # - CFME does not currently support client ssl certificate for authentication
 export QPID_SSL_CERT_DB=/etc/qpid/certs
 
+# This is needed only if we are using Apache from SCL which is installed into
+# a different directory
+scl_apache=1
+if [ -n "$scl_apache" ] && [ ! -h /opt/rh/httpd24/root/var/www/html ]
+then
+  mv /opt/rh/httpd24/root/var/www/html /opt/rh/httpd24/root/var/www/html_24
+  ln -s /var/www/html /opt/rh/httpd24/root/var/www/html
+fi
+
 [[ -s /etc/default/evm_bundler ]] && source /etc/default/evm_bundler
 [[ -s /opt/rh/postgresql92/enable ]] && source /opt/rh/postgresql92/enable
 [[ -s /opt/rh/nodejs010/enable ]] && source /opt/rh/nodejs010/enable
-[[ -s /etc/default/evm_productization ]] && source /etc/default/evm_productization
+[[ -s /opt/rh/httpd24/enable ]] && source /opt/rh/httpd24/enable
+export PATH=$PATH:/opt/rubies/ruby-2.0.0-p598/bin
```

Comment:
This feels wrong.  I thought we set this up via the SCL in etc/default/evm_bundler or perhaps should be done above like the other 2 /enable scripts.  Aside, @jvlcek hit something similar yesterday, and I don't understand why it's not already available at this point